frontend/src/components/prompt/PopoverPrompt.vue

```diff
@@ -0,0 +1,184 @@
+<template>
+  <v-menu
```

Comment:
I really love the way this looks. You outdid yourself with this one :) I also tried to break it on ridiculously tiny screen sizes, but it was always useable.